core/src/main/scala/cats/instances/queue.scala

```diff
@@ -0,0 +1,145 @@
+package cats
+package instances
+
+import cats.syntax.show._
+
+import scala.annotation.tailrec
+import scala.collection.immutable.Queue
+import scala.util.Try
+
+trait QueueInstances extends cats.kernel.instances.QueueInstances {
+
+  implicit val catsStdInstancesForQueue: Traverse[Queue] with Alternative[Queue] with Monad[Queue] with CoflatMap[Queue] =
```

Comment:
A bunch of these overrides are not tested. I am not sure how big a deal it is since they don't seem to be tested for the `list` instance either.